internal/controller/components/feastoperator/monitoring/feastoperator-prometheusrules.tmpl.yaml

```
@@ -0,0 +1,98 @@
+            annotations:
+              message: 'High error budget burn for {{`{{`}}$labels.instance{{`}}`}} (current value: {{`{{`}}$value{{`}}`}} ).'
+              summary: Feast Operator Probe Success Burn Rate
+            expr: |
+              sum(probe_success:burnrate5m{instance=~"feast-operator-controller-manager"}) by (instance) > (14.40 * (1-0.98000))
+              and
+              sum(probe_success:burnrate1h{instance=~"feast-operator-controller-manager"}) by (instance) > (14.40 * (1-0.98000))
+            for: 2m
+            labels:
+              severity: critical
+              instance: feast-operator-controller-manager
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)